### DISCUSSION TITLE
Triple Clicking Text Highlights Entire String

### DISCUSSION THREAD

**Discussion by 19peytonsmith** [ppy/osu#27267] [Ideas] [answered]
![image](https://github.com/ppy/osu/assets/54728307/dfaf8a7d-046a-4bce-ab1f-ae20e4a0ddd1)

I noticed for any text field really that when you triple click it only highlights that one word and toggles between highlighting it and not. It's pretty standard now that triple-clicking it highlights the entire thing, would be a good QOL change.

**Comment by Joehuu** [comment, marked as answer]
Relevant framework-side issue: https://github.com/ppy/osu-framework/issues/4974
